pegasus/sites.v3/code.org/views/display_video_thumbnail.haml

```diff
@@ -3,11 +3,34 @@
 - download_filename ||= nil
 - download_path ||= nil
 - caption ||= nil
+- width ||= '100%'
+- image ||= "//i.ytimg.com/vi/#{video_code}/0.jpg"
+- play_button ||= 'center' # 'center' or 'caption'
 
-.videothumbnail{:onclick => "return showVideo_#{id}()", :style => "cursor:pointer"}
-  %div{:style => "position:relative; overflow:hidden"}
-    %img{:src => image, :width => "60%"}
-    %img{:src => "/shared/images/play-button.png", :style => "position:absolute; top:100px; left:30%", :width => "40"}
-  .video_caption_link= caption
+:scss
+  .play-button-caption .play {
+    display: none;
+  }
+  .play-button-caption .thumbnail-wrapper .video_caption_link {
+    position: absolute;
+    bottom: 0;
+    height: 50px;
+    color: white;
+    background-color: black;
+    line-height: 50px;
+    width: 100%;
+    opacity: 0.7;
+    padding-left: 50px;
+    background-image: url(/shared/images/play-button.png);
+    background-size: 40px 40px;
+    background-repeat: no-repeat;
+    background-position: 5px 5px;
+  }
+
+.videothumbnail{:onclick => "return showVideo_#{id}()", :style => "cursor:pointer; width: #{width}", class: "play-button-#{play_button}"}
+  %div.thumbnail-wrapper{:style => "position:relative; overflow:hidden;"}
+    %img.thumbnail-image{:src => image, style: 'width: 100%'}
+    %img.play{:src => "/shared/images/play-button.png", :style => "position:absolute; top:30%; left:40%", :width => "20%"}
```

Comment:
yeah, I tried something like top:40% left 40% and width 20%, but I think we do want the play element to be proportionally bigger for smaller thumbnails..